### ISSUE TITLE
BM25 normalization

### ISSUE THREAD

**Issue by melisabok** [openreview/openreview-expertise#55]
> it appears that BM25 normalizes the scores within each submission, that is, for each paper there is a reviewer with similarity 1. Of course, it does not change the ranking for the paper, but is there a way to rank the papers in order of decreasing similarity with a given reviewer?

**Comment by carlosmondra** [comment]
TL;DR: No, it's not possible to rank papers given a reviewer using BM25.

What you are saying is correct. The normalization occurs across all reviewers for a single paper. The normalization is done so that the results are easier to interpret. We realize that the paper1-reviewer1 score compared to paper2-reviewer2 is not comparable even if both of them have score of 1. Even if they were NOT normalized, a score of 10 for paper1-reviewer1 would not be comparable to a score of 10 for paper2-reviewer2.

In this case, we aim for the papers to have the best possible reviewers and not the other way around. This way we can compare paper1-reviewer1's score against paper1-reviewer2's score. Therefore, we avoid having papers with very low scores for every reviewer.

**Comment by IvanStelmakh** [comment]
Thank you for clarifying this!

Does the same thing hold for ELMO? More generally, what method would you recommend (if any) to rank papers for a given reviewer?

**Comment by carlosmondra** [comment]
This is only true for BM25. You can use ELMo scores without normalization so that you can compare the scores. The output will sort the scores by paper id and then by the score that the paper has with their reviewer, but you can postprocess the result so that you sort them by reviewer and then by score.

If you ask me for what method you should use, I would recommend SPECTER+MFR which is far superior than ELMo and BM25. We have had very positive feedback from PCs when using SPECTER+MFR.

**Comment by carlosmondra** [comment]
Closing this issue as the question was already answered.
